src/assets/i18n/fr.json

```diff
@@ -417,6 +417,17 @@
         "title": "Télémétrie",
         "label": "Envoyer des données d'utilisation anonymes",
         "description": "Les données envoyées seront limitées à la version d'Oyasumi que vous utilisez, et au langage paramétré. Aucune autre donnée n'est envoyée, et ces dernières sont complètement anonymes. Ces données sont utilisées à des fins statistiques."
+      },
+      "systemTray": {
+        "title": "Barre d'état",
+        "close": {
+          "label": "Fermer l'application dans la barre d'état",
+          "description": "Réduit l'application dans la barre d'état système lorsque vous cliquez sur le bouton 'Fermer'"
+        },
+        "start": {
+          "label": "Démarrer dans la barre d'état",
+          "description": "Démarrer l'application cachée dans la barre d'état. Vous pouvez la restaurer en cliquant sur l'icône dans la barre d'état."
+        }
```

Comment:
Maybe these need to be updated based on the previous changes to the English translations?